dashboard/app/models/concerns/concept_difficulties.rb

```diff
@@ -0,0 +1,20 @@
+module ConceptDifficulties
+  extend ActiveSupport::Concern
+
+  CONCEPTS = %w(
+    sequencing
+    debugging
+    repeat_loops
+    repeat_until_while
+    for_loops
+    events
+    variables
+    functions
+    functions_with_params
+    conditionals
```

Comment:
We should discuss (possibly with Alice, Ryan, and/or Kiki) whether we want to tag levels with non-CS concepts [as was done in the manual tagging]. For example, we may or may not want to add outside_difficulty or geometry or otherwise.